File: logica-de-programacao/secao-10/exercicio/retangulo.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    double base, altura, area, perimetro, diagonal;

    cout << "Base do retangulo: ";
    cin >> base;

    cout << "Altura do retangulo: ";
    cin >> altura;

    cout << fixed << setprecision(4);

    area = base * altura;
    cout << "AREA = " << area << endl;

    perimetro = (base * 2) + (altura * 2);
    cout << "PERIMETRO = " << perimetro << endl;

    diagonal = sqrt(pow(base, 2) + pow(altura, 2));
    cout << "DIAGONAL = " << diagonal << endl;
    
    return 0;
}

File: logica-de-programacao/secao-11/exercicios/retangulo.java
package exercicios;

import java.util.*;

public class retangulo {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        System.out.print("Base do retangulo: ");
        double base = sc.nextDouble();

        System.out.print("Altura do retangulo: ");
        double altura = sc.nextDouble();

        double area = base * altura;
        double perimetro = 2 * (base + altura);
        double diagonal = Math.sqrt(Math.pow(base, 2) + Math.pow(altura, 2));

        System.out.println("AREA = " + String.format("%.4f", area));
        System.out.println("PERIMETRO = " + String.format("%.4f", perimetro));
        System.out.println("DIAGONAL = " + String.format("%.4f", diagonal));

        sc.close();
    }
}
